shelley/chain-and-ledger/formal-spec/incentives.tex

```diff
@@ -0,0 +1,572 @@
+\end{figure}
+
+\noindent
+The gross reward that is received by the Stakepool is, on average, directly proportional to the
+stake that it controls, as a proportion of the total \ada{} that is in circulaton.
+It is limited by the rewards limit fraction.  Once the pool operator costs are subtracted,
+the net reward is then distributed to the owner(s) and delegators in proportion to the
+stake that each group controls.
+
+\subsubsection*{The Rewards that are received by each Delegator}
+
+\begin{figure}[h!]
+\begin{center}
+\begin{tabular}{||l|p{6cm}|l||}
+  \hline \hline
+\textbf{Parameter} & \textbf{Definition} & \textbf{Calculated as} \\\hline
+$SP_T$ & All \ada{} that is controlled by the Stakepool & $SP_p + SP_d$ \\\hline
+$R_D$ & The Total Reward to the Delegators & $R_N - R_O$ \\\hline
+$D_S$ & The \ada{} that is delegated by  $D$ & \\\hline
+$D_\%$ & The proportion of \ada{} in the pool that is delegated by $D$ & \\\hline
+$D_R$ & The \ada{} rewards that are received by $D$ & $R_N \times R_O$ \\\hline
+\hline
+\end{tabular}
+\end{center}
+\caption{The Rewards that are received by a specific Delegator, $D$.}
+\end{figure}
+
+\noindent
+The rewards that are received by an individual delegator are directly proportional to the
+stake that the delegator has placed in the Stakepool, based on the net rewards.
+
+\subsubsection*{The Rewards that are received by each Owner}
+
+\begin{figure}[h!]
+\begin{center}
+\begin{tabular}{||l|p{6cm}|l||}
+  \hline \hline
+\textbf{Parameter}  & \textbf{Definition} & \textbf{Calculated as} \\\hline
+$R_O$ & The Total Reward to the Owner(s) & $R_N \times SP_m $ \\\hline
+$N$ & The number of Owners & \\\hline
+$O_R$ & The \ada{} rewards that are received by each owner & $R_O \div N$ \\\hline
+\hline
+\end{tabular}
+\end{center}
+\caption{The Rewards that are received by each Owner.}
+\end{figure}
+
+\noindent
+In the Incentivised Testnet, the owner rewards are divided equally among the
+owners, regardless of the proportion of stake that they pledge.  The assumption
+is that, in practice, each owner will pledge an equal amount of stake, but this
+is not enforced by the system.
+
+\clearpage
+\subsection{Calculating Rewards in External Currencies}
+
+\begin{figure}[h!]
+\begin{center}
+\begin{tabular}{||l|p{6cm}|l||}
+  \hline \hline
+\textbf{Value} & \textbf{Description} & \textbf{Calculated as} \\\hline
+$R_A$ &  Rewards value in \ada{} &\\\hline
+$D$ &  Dollar Price per \ada{} &\\\hline
+$R_D$ &  Rewards value in dollars & $R_A \times D$ \\\hline
+\hline
+\end{tabular}
+\end{center}
+\caption{Calculating Monetary Rewards}
+\label{fig:monetary}
+\end{figure}
+
+\noindent
+As usual, rewards in \ada{} can easily be converted to a dollar or other equivalent using
+the current exchange rate, $D$, as shown % in Figure~\ref{fig:monetary}
+above.  For example, if the rewards are \ADA{10,000} and the exchange rate is
+\$0.04 (1 \ada{} is worth 4 cents).  Realising this value would require an exchange,
+of course.
```

Comment:
in the later figure, you are using 0.039 cents, not a problem, but why not stick to the same amount here ?